os_brick/initiator/connectors/lightos.py

```diff
@@ -71,7 +88,9 @@ def get_connector_properties(root_helper, *args, **kwargs):
                                              execute=kwargs.get('execute'))
         hostnqn = utils.get_host_nqn()
         found_dsc = lightos_connector.find_dsc()
-
+        host_ips = lightos_connector.get_ip_addresses()
+        LOG.info('Host current IP(s) are ',host_ips)
```

Comment:
Add which host, it will help debugging later.

Reply:
@muliby-lb @yuval-lb 
Interestingly netutil.py  is also using package netifaces, which means we should not have issues using netifaces. 
I'm using netutil.py for IPv4/6 validity checks. Current code will read all available IPs and pass-on to cinder. 
If IPv6 is not enabled on host then this function  will read only read IPv4 addresses.  
It will not be a bad idea to move forward with all IPs and then apply filtering on undesired IPs once everything works. 

@sagi-lb  Is there a way to determine if an IP belongs to a physical interface vs virtual.